src/testing/loadtest-azure/scripts/file-upload-to-loadtest.ps1

```diff
@@ -17,7 +17,13 @@ param
   [string] $testFileName,
   
   # Test File ID is auto-generated when not set (default)
-  [string] $testFileId = (New-Guid).toString()
+  [string] $testFileId = (New-Guid).toString(),
+
+  # if set to true script will set pipeline variables
+  [bool] $pipeline = $false,
+
+  # if set to true script will wait till file was validated
+  [bool] $wait = $false 
```

Comment:
Is there a scenario where we *don't* want to wait for the validation? I would think that it doesn't work without it, does it?

Reply:
My thinking was that this is only needed when the file upload is embedded into the pipeline. And it might not be needed when other files (other than filetype 0 for jmx) are uploaded. I haven't verified that yet as we do not upload any other files at the moment. https://learn.microsoft.com/en-us/rest/api/loadtesting/dataplane/test/get-test-file?tabs=HTTP#filetype

Reply:
I see, thanks. How about setting the default to `$true` for now, as it's unclear whether the validation ever doesn't happen?

Reply:
```suggestion
  [bool] $wait = $true
```

ok for me